frontend/src/components/navbar.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "../styles/navbar.css";
+
+function Navbar(): React.ReactElement {
```

Comment:
Please rename the file as `Navbar.tsx` Naming conventions for react `.tsx` files should always begin with capital letter

Reply:
Can you also put Navbar to its own folder called `Navbar` just for organization sake